src/motions.c

```diff
@@ -51,13 +64,17 @@ bool process_motions(uint16_t keycode, const keyrecord_t *record, uint16_t qk_mo
             register_motion(qk_mods | VIM_L, record);
             break;
         case KC_B:
+#ifndef VIM_FOR_ALL
```

Comment:
Just a small thing, I think I would prefer if these were just ripped out from this switch statement. So rather than having to ifdef around all of them, we can just have `VIM_REDUCE_MAC_NOMAC_TO_CONST` use a default impl that just does nothing (much like `DO_NUMBERED_ACTION`).

Reply:
Done, let me know if that looks good to you.
I also noticed I forgot a bunch of breaks in the new switch statement so I fixed that as well.